src/elements/content-sidebar/SidebarContent.js

```diff
@@ -33,6 +34,7 @@ const SidebarContent = ({ actions, children, className, elementId, sidebarView,
                 {title && <h3 className="bcs-title">{title}</h3>}
                 {actions}
             </div>
+            {subheader && <div className="bcs-content-subheader">{subheader}</div>}
```

Comment:
I see we pass in children below this. Why do we not use the `children` prop instead? Is the intention for subheader supposed to not be scrollable? Do you think we have enough usecases for a subheader prop? Just want to make sure we are not getting into the habit of bloating the SidebarContent component